python/huggingfaceserver/README.md

```diff
@@ -92,6 +93,7 @@ spec:
       - --model_id=bert-base-uncased
       - --predictor_protocol=v2
       - --tensor_input_names=input_ids
+      - --disable_vllm
```

Comment:
I think we probably we do not want to confuse user that they have to set this configure to get Bert work. I think a better example would be the case that vllm is default used by show an example that it can be disabled.

Reply:
I have added a disable vllm example

Reply:
I just trying out the new huggingface runtime and notice this. I feel like a bit werid to have vllm as default runtime (as the PR mentioned vllm as alternative backend).

how about `--backend=vllm`, having `huggingface` as default and vllm as alternative and future can support other backend like `tensorRT`

what do you think? @yuzisun